pages/mobile/base.py

```diff
@@ -14,6 +14,7 @@ class Base(Page):
     _body_locator = (By.TAG_NAME, 'body')
     _menu_items_locator = (By.CSS_SELECTOR, 'nav a')
     _menu_button_locator = (By.ID, 'menu-button')
+    _mobile_header = (By.CSS_SELECTOR, 'header.slide-on-exposed')
```

Comment:
You've added the code in this file to both `base.py` and `home.py`, so please remove it from one of these as it isn't needed in both places. I suggest leaving it here, as it is applicable to all mobile pages.